ecs-cli/modules/cli/logs/logs_app.go

```diff
@@ -309,6 +309,9 @@ func logConfigMisMatchError(taskDef *ecs.TaskDefinition, fieldName string) error
 // CreateLogGroups creates any needed log groups for the task definition to use CloudWatch Logs
 func CreateLogGroups(taskDef *ecs.TaskDefinition, logClientFactory cwlogsclient.LogClientFactory) error {
 	for _, container := range taskDef.ContainerDefinitions {
+		if container.LogConfiguration == nil || container.LogConfiguration.LogDriver == nil || aws.StringValue(container.LogConfiguration.LogDriver) != "awslogs" {
+			continue
+		}
 		logConfig, err := getContainerLogConfig(container)
 		if err != nil {
 			return err
```

Comment:
will this still fail if NO containers use awslogs? (and if so, can we add a test for it?)

Reply:
It won't fail if no containers use awslogs- in that case nothing happens. Normally if log groups are created, then you'll get an Info message about it, if no containers use awslogs, then you get no message. I guess using `--create-log-groups` when no containers use awslogs is a user error- but I don't really want that to cause the command to fail. What do you think?

Reply:
I don't think it should fail either, though it feels like we may want to `WARN` in this case so customers know that option isn't doing anything. Would adding that check blow up the scope of the current change? If not let's add it, if so I don't consider it a blocker.